pkgs/development/tools/glualint/default.nix

```diff
@@ -0,0 +1,75 @@
+{ mkDerivation
+, aeson
+, base
+, bytestring
+, containers
+, deepseq
+, directory
+, effectful
+, fetchFromGitHub
+, filemanip
+, filepath
+, lib
+, MissingH
+, optparse-applicative
+, parsec
+, pretty
+, signal
+, uu-parsinglib
+, uuagc
+, uuagc-cabal
+}:
+mkDerivation rec {
+  pname = "glualint";
+  version = "1.24.3";
+
+  src = fetchFromGitHub {
+    owner = "FPtje";
+    repo = "GLuaFixer";
+    rev = version;
+    hash = "sha256-FmgQYONm1UciYa4Grmn+y1uVh9RSL90hO2rWusya0C0=";
+  };
+
+  isLibrary = true;
+  isExecutable = true;
+  libraryToolDepends = [ uuagc uuagc-cabal ];
+  libraryHaskellDepends = [
+    aeson
+    base
+    bytestring
+    containers
+    MissingH
+    parsec
+    pretty
+    uu-parsinglib
+    uuagc
+    uuagc-cabal
+  ];
+  executableHaskellDepends = [
+    aeson
+    base
+    bytestring
+    containers
+    deepseq
+    directory
+    effectful
+    filemanip
+    filepath
+    optparse-applicative
+    signal
+  ];
+
+  preBuild = ''
+    echo "Generating attribute grammar haskell files"
+    uuagc --haskellsyntax --data src/GLua/AG/AST.ag
```

Comment:
It appears that the `cabal sdist` command I'll use to upload glualint will already turn the UUAGC `.ag` files into `.hs` files. 

When `glualint` is uploaded to hackage, I'm hoping that the Nixpkgs hackage script will automatically pick it up and add it to `pkgs/development/haskell-modules/hackage-packages.nix`, making this entire file no longer needed.